docusaurus/docs-internals/editor-controls/multivalue-controls/margin.md

```diff
@@ -187,3 +244,145 @@ When the `edges` is `vertical`, it will enable actions for `y` axis.
   }
 }
 ```
+
+#### Default value examples
+
+In this example, the `margin` control is defined with several default values that specify its initial state. The `id` and `type` properties both indicate that this control deals with `margin` settings. The `default` object within the control contains the properties and their default values.
+
+```js
+{
+  id: "margin", 
+  type: "margin",
+  default: {
+    type: "ungrouped",
+    value: 10,
+    unit: "px",
+    top: 0,
+    topUnit: "px",
+    right: 0,
+    rightUnit: "px",
+    bottom: 0,
+    bottomUnit: "px",
+    left: 0,
+    leftUnit: "px"
```

Comment:
check one more time keys in `Margin/converters.ts` on `fromElementModel`

as example instead of `bottomUnit` should be `bottomSuffix`